src/vm/i386/asmhelpers.asm

```diff
@@ -118,6 +118,23 @@ EXTERN @ProfileEnter@8:PROC
 EXTERN @ProfileLeave@8:PROC
 EXTERN @ProfileTailcall@8:PROC
 
+COMPlusFrameHandler proto c
```

Comment:
This is part of the file that has just macros and forward declarations that do not generate any code. It would look better to have it later in the file - e.g. before  `CallRtlUnwind` may be a good place.